### ISSUE TITLE
Ionic info doesn't show updated Android SDK Tools

### ISSUE THREAD

**Issue by Jun711** [ionic-team/ionic-cli#2752]
**Description:**

I updated my Android SDK Tools via Android studio. My Android Studio shows that Android SDK Tools version 26.1.1 is installed but with ionic info, I see the following. Android SDK Tools at 24.4.1. 
1) Could it be my env path issue [source code](https://github.com/ionic-team/ionic-cli/blob/1d0f21c3bad7b1214c8e7bd35a34adb492478c51/packages/%40ionic/cli-utils/src/lib/android.ts) 

**My `ionic info`:**
```cli packages: (/Users/user/.nvm/versions/node/v8.2.1/lib/node_modules)

    @ionic/cli-utils  : 1.10.2
    ionic (Ionic CLI) : 3.10.3

global packages:

    Cordova CLI : 6.5.0 

local packages:

    @ionic/app-scripts : 2.1.4
    Cordova Platforms  : android 6.2.3 ios 4.4.0
    Ionic Framework    : ionic-angular 3.6.1

System:

    Android SDK Tools : 24.4.1
    ios-deploy        : 1.9.2 
    ios-sim           : 6.0.0 
    Node              : v8.2.1
    npm               : 5.2.0 
    OS                : macOS Sierra
    Xcode             : Xcode 8.3.2 Build version 8E2002 

**My `sdkmanager info`:**

Installed packages:=====================] 100% Computing updates...             
  Path                        | Version | Description                    | Location                    
  -------                     | ------- | -------                        | -------                     
  build-tools;25.0.2          | 25.0.2  | Android SDK Build-Tools 25.0.2 | build-tools/25.0.2/         
  build-tools;26.0.1          | 26.0.1  | Android SDK Build-Tools 26.0.1 | build-tools/26.0.1/         
  emulator                    | 26.1.4  | Android Emulator               | emulator/                   
  extras;android;m2repository | 47.0.0  | Android Support Repository     | extras/android/m2repository/
  extras;google;m2repository  | 58      | Google Repository              | extras/google/m2repository/ 
  patcher;v4                  | 1       | SDK Patch Applier v4           | patcher/v4/                 
  platform-tools              | 26.0.0  | Android SDK Platform-Tools     | platform-tools/             
  platforms;android-24        | 2       | Android SDK Platform 24        | platforms/android-24/       
  platforms;android-25        | 3       | Android SDK Platform 25        | platforms/android-25/       
  tools                       | 26.1.1  | Android SDK Tools              | tools/

**Comment by dwieeb** [comment]
I think you have two SDKs installed. The CLI uses the environment variable `$ANDROID_HOME` to locate it, as you saw. I think Cordova uses it, as well, but not 100% sure. 

Which SDK does it use when you run `ionic cordova build android`?

**Comment by Jun711** [comment]
@dwieeb , right, I think I have two SDKs installed. one via Android Studio and one via brew cask. 

I have fixed it by setting my ANDROID_HOME variable to point to the sdk I want. IIRC, it showed  24.4.1 when I ran ionic cordova build android. 
But, now, when I run ionic cordova build android, it shows that ANDROID_HOME pointing to the sdk I set in my profile. 

On the other hand, does ionic cordova cli work with android-sdk 26? 
I am using android-sdk 25.2.3 now. It complained that there is no template folder when I used android-sdk 26.
